contracts/staking/Staking.sol

```diff
@@ -1237,16 +1245,7 @@ contract Staking is StakingV1Storage, GraphUpgradeable, IStaking {
         }
 
         // When there are tokens to claim from the rebate pool, transfer or restake
-        if (tokensToClaim > 0) {
-            // Assign claimed tokens
-            if (restake) {
-                // Restake to place fees into the indexer stake
-                _stake(alloc.indexer, tokensToClaim);
-            } else {
-                // Transfer funds back to the indexer
-                require(graphToken.transfer(alloc.indexer, tokensToClaim), "!transfer");
-            }
-        }
+        _sendRewards(graphToken, tokensToClaim, alloc.indexer, restake);
```

Comment:
This means the indexer agent has to be updated too right? Since this is only for query fees, I think its ok to not have this feature in the agent for now

Reply:
So to not change the current claim() interface, if you pass this method restake=true it will restake if restake=false it will send rewards out.